src/components/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { SyntheticEvent, useState, useMemo } from 'react';
+import Autocomplete from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import { CircularProgress } from '@mui/material';
+import { debounce } from 'lodash';
+import { useSnackbar } from 'notistack';
+import FetchGeoData from 'components/FetchGeoData';
+import { CITIES } from './citiesList';
+import * as styles from './styles';
```

Comment:
Import it like `import * as classes from './styles'`
Use it in this way - `css={classes.foo}`
Check TS support for `emotion`